scripts/models/qwen3-4B-Instruct.sh

```diff
@@ -0,0 +1,17 @@
+MODEL_ARGS=(
```

Comment:
It's suggested to source the qwen3-4B file and append the `--rotary-base 5000000`